forge-gui/res/cardsfolder/upcoming/grell_philosopher.txt

```diff
@@ -0,0 +1,12 @@
+Name:Grell Philosopher
+ManaCost:2 U
+Types:Creature Horror Wizard
+PT:1/4
+K:ManaConvert:U->AnyColor:You may spend blue mana as though it were mana of any color to pay the activation costs of CARDNAME's abilities.
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigEffect | TriggerDescription$ Aberrant Tinkering — When CARDNAME enters the battlefield and at the beginning of your upkeep, each Horror you control gains all activated abilities of target artifact an opponent controls until end of turn.
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | Execute$ TrigEffect | TriggerZones$ Battlefield | Secondary$ True | TriggerDescription$ Aberrant Tinkering — When CARDNAME enters the battlefield and at the beginning of your upkeep, each Horror you control gains all activated abilities of target artifact an opponent controls until end of turn.
+SVar:TrigEffect:DB$ Effect | ValidTgts$ Card.Artifact+OppCtrl | TgtZone$ Battlefield | TgtPrompt$ Select target artifact an opponent controls | StaticAbilities$ STSteal | RememberObjects$ Targeted
+SVar:STSteal:Mode$ Continuous | Affected$ Creature.Horror+YouCtrl | EffectZone$ Command | GainsAbilitiesOfDefined$ RememberedLKI
```

Comment:
Which Horrors it affects should probably be locked in when the trigger resolves

Might need to use Imprint too

Reply:
Please try like this:

```
SVar:TrigEffect:DB$ Effect | ValidTgts$ Card.Artifact+OppCtrl | TgtZone$ Battlefield | TgtPrompt$ Select target artifact an opponent controls | StaticAbilities$ STSteal | ImprintCards$ Targeted | RememberObjects$ Valid Creature.Horror+YouCtrl | ForgetOnMoved$ Battlefield
SVar:STSteal:Mode$ Continuous | Affected$ Card.IsRemembered | EffectZone$ Command | GainsAbilitiesOfDefined$ ImprintedLKI
```

Reply:
Much ass, grassy ass again, mate!!!

Reply:
And don't I need a SubAbility$ DBCleanup
SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True 
for the last 2 lines? 
And is there a ClearImprinted$ True parameter? If so, is it needed?

Reply:
Nah not needed in this case

you're only Remembering/Imprinting the stuff on an Effect card

that will get cleaned up naturally once it ends

Reply:
O I C.